src/login.js

```diff
@@ -0,0 +1,87 @@
+import { getEmailErrorMsg, getPasswordErrorMsg, getNicknameErrorMsg } from './validationErrorMessage.js';
+
+// Elements
+const emailInput = document.querySelector('#email');
+const emailError = document.querySelector('#email + .input-validation-error');
+
+const passwordInput = document.querySelector('#password');
+const passwordError = document.querySelector('.password-container + .input-validation-error');
+
+const submitButton = document.querySelector('.btn-login');
+
+const visibleIcon = document.querySelector('.visible-icon');
+const invisibleIcon = document.querySelector('.invisible-icon');
+
+// State tracking
+let isEmailValid = false;
+let isPasswordValid = false;
+
+// 에러 표시를 업데이트하는 함수
+function updateErrorDisplay(inputElement, errorElement, showError) {
+  if (showError) {
+    inputElement.classList.add('error');
+    errorElement.style.visibility = 'visible';
+  } else {
+    inputElement.classList.remove('error');
+    errorElement.style.visibility = 'hidden';
+  }
```

Comment:
P3:
아래 코드도 동일 동작합니다.

```suggestion
  if (showError) {
    inputElement.classList.add('error');
    errorElement.style.visibility = 'visible';
    return;
  }
  
  inputElement.classList.remove('error');
  errorElement.style.visibility = 'hidden';
```